subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/GrettySmokeTest.groovy

```diff
@@ -32,7 +38,7 @@ class GrettySmokeTest extends AbstractPluginValidatingSmokeTest {
                 id "org.gretty" version "${TestedVersions.gretty}"
             }
 
-            ${mavenCentralRepository()}
+            ${jcenterRepository()}
```

Comment:
:o: Did you mean this change? It feel silly with the JCenter shutdown

Reply:
Yes, Gretty is currently only on JCenter. They are working to release their first version to Maven central, though that didn't happen, yet.